### DISCUSSION TITLE
Lsp Info autostart

### DISCUSSION THREAD

**Discussion by edshamis** [LazyVim/LazyVim#466] [Q&A] [answered]
What does autostart means in Lsp Info and where/how it's configured?

**Comment by folke** [comment, marked as answer]
Check the lspconfig docs.

**Reply by edshamis** [reply]
I'm confused by the fact that for example null-ls shows autostart false in Lsp Info but it is still autostarted when a file is opened.

**Reply by folke** [reply]
that's just how null-ls works. It has autostart=false, because null-ls has their own starting logic. Check the null-ls docs to learn more
